packages/helpers/src/category/getCategoryRoute.ts

```diff
@@ -0,0 +1,50 @@
+import { Category } from "@shopware-pwa/types";
+import { getCategoryUrl } from "./getCategoryUrl";
+type LinkedCategory = Pick<
+  Category,
+  "type" | "externalLink" | "seoUrls" | "internalLink" | "id" | "linkType"
+>;
+
+/**
+ * Returns category or navigation URL and route informations for resolving SEO url.
+ * Use it with combination of <RouterLink> or <NuxtLink> in Vue.js or Nuxt.js projects.
+ *
+ * Example:
+ * ```html
+ * <RouterLink :to="getCategoryRoute(navigationElement)">
+ * ```
+ */
```

Comment:
maybe we can categorize the method to some existing groups: https://frontends.shopware.com/packages/helpers.html (see the side navi)